### DISCUSSION TITLE
What ports are required?

### DISCUSSION THREAD

**Discussion by OptimoSupreme** [mikebrady/shairport-sync#1823] [Q&A] [answered]
I've opened up the ports I can find in the config as well as mdns, but I cannot stream music unless I disable Firewalld in which case it works fine. My current config:

```
$ sudo firewall-cmd --list-all
public (default, active)
  target: default
  ingress-priority: 0
  egress-priority: 0
  icmp-block-inversion: no
  interfaces: enu1u1u1
  sources: 
  services: dhcpv6-client mdns ssh
  ports: 5000/tcp 6001-6011/udp 5353/udp      #important bits
  protocols: 
  forward: yes
  masquerade: no
  forward-ports: 
  source-ports: 
  icmp-blocks: 
  rich rules: 
```

**Comment by mikebrady** [comment, marked as answer]
Thanks for the post. This is dealt with in the [TROUBLESHOOTING.md](https://github.com/mikebrady/shairport-sync/blob/master/TROUBLESHOOTING.md#ufw-firewall-blocking-ports-commonly-includes-raspberry-pi) guide.

**Reply by OptimoSupreme** [reply]
Ah shoot, I’m that guy now!! Thank you for the response.
